pkg/node/devnode.go

```diff
@@ -110,6 +113,18 @@ func NewDevBee(logger logging.Logger, o *DevOptions) (b *DevBee, err error) {
 		return nil, fmt.Errorf("eth address: %w", err)
 	}
 
+	var authenticator *auth.Authenticator
+
+	username := os.Getenv("BEE_AUTH_USERNAME")
+	password := os.Getenv("BEE_AUTH_PASSWORD")
```

Comment:
The node package does not handle env variables, it explicitly expect input arguments to functions. Currently all configuration is done in the cmd/bee package and I would suggest to make the same with these parameters also. Just adding string flags as with any other configuration parameters will also provide the possibility to configure them with env variables, the same way as with any other parameter.